pandas/io/excel/_base.py

```diff
@@ -755,6 +755,16 @@ class ExcelWriter(metaclass=abc.ABCMeta):
     >>> with ExcelWriter("path_to_file.xlsx", mode="a", engine="openpyxl") as writer:
     ...     df.to_excel(writer, sheet_name="Sheet3")
 
+    You can specify arguments to the underlying engine. For example to not
+    calculate the result of a formula:
+
+    >>> df = pd.DataFrame(["=1+1"])
+    ... with ExcelWriter(
+    ...     "path_to_file.xlsx",
+    ...     engine_kwargs={"strings_to_formulas":False}
+    ... ) as writer:
```

Comment:
I think this will only be valid with xlsxwriter, is that right? Can you specify the engine here to make it clear that's what is being used.